packages/hadron-type-checker/src/type-checker.js

```diff
@@ -23,7 +23,7 @@ const {
   Code,
   BSONSymbol,
   Timestamp,
-  BSONMap
```

Comment:
`BSONMap` is not a thing in bson@4 but also driver never returns this class explicitly anyway, so everything worked fine. Also worth noting that selecting this type is broken in document view anyway for a while now (I'll open a ticket for that, alternatively we can also consider just removing it completely)